packages/compass-schema/src/components/field/field.jsx

```diff
@@ -110,6 +112,11 @@ class Field extends Component {
     return null;
   }
 
+  async componentDidMount() {
+    const { enableMaps } = await preferencesIpc.getPreferences();
+    this.setState({ enableMaps });
+  }
```

Comment:
Seems like we only set the value once, but don't update it if it was changed. There probably should be a corresponding subscription to preferences change here?

Reply:
@gribnoysup We decided to leave that out of this PR for now, since it didn’t change from how this was behaving before

Reply:
Gotcha, thanks for the context. Let's open a ticket for that if we don't have one yet, but otherwise that answers my question

Reply:
https://jira.mongodb.org/browse/COMPASS-6127